src/codecs/bib/__file_snapshots__/article.bib

```diff
@@ -1,9 +1,9 @@
-@article{Smith2010title,
+@article{Smith2009title,
 	title = {The title of the article},
 	author = {Smith, Jane and Jones, Jim},
-	date = {2010-01-01},
-	year = {2010},
-	month = {1},
-	day = {1},
+	date = {2009-12-31},
```

Comment:
This is related to timezone differences between machines. I thought I had resolved it in the `csl` codec (or maybe the `date` codec) but obviously not. Could you please (a) revert this change so that tests pass on master when we merge this (b) raise an issue about this.